compositing-2/Overview.bs

```diff
@@ -709,16 +709,15 @@ If the 'knock-out' property is disabled, each element within the group is only c
 -->
 <h3 id="pagebackdrop">The Page Group</h3>
 
-The <a href="#isolatedgroups">isolated group</a for the root element is the
+The <a href="#isolatedgroups">isolated group</a> for the root element is the
 page group. All other elements and groups are composited into this group.
 The background of the root element (if specified) is painted into the page
 group, and any filter, clip-path, mask and and opacity is then applied, before
 compositing into the <a href="#rootgroup">root group</a>.
 
 <h3 id="rootgroup">The Root Group</h3>
 The root group is the final compositing output, and is is above the page
-group. Compositing of the page group into the root group is performed with the
-specified blend mode of the root element.
+group.
 
 <p class="note">Browsers often use an infinite white, 100% opaque root group,
```

Comment:
css-color defines this. Maybe add a callout to that section?

Reply:
Done. I linked to https://www.w3.org/TR/css-color-3/#sample. Is this what you had in mind?

Reply:
This is the link I was suggesting: https://www.w3.org/TR/css-color-4/#sample

Reply:
Updated.